liquibase-couchbase/src/test/java/liquibase/ext/couchbase/executor/CouchbaseExecutorTest.java

```diff
@@ -0,0 +1,79 @@
+package liquibase.ext.couchbase.executor;
+
+import com.couchbase.client.java.Cluster;
+import liquibase.exception.DatabaseException;
+import liquibase.ext.couchbase.database.CouchbaseConnection;
+import liquibase.ext.couchbase.database.CouchbaseLiquibaseDatabase;
+import liquibase.ext.couchbase.exception.StatementExecutionException;
+import liquibase.ext.couchbase.executor.service.TransactionExecutorService;
+import liquibase.ext.couchbase.statement.CouchbaseStatement;
+import liquibase.ext.couchbase.statement.CouchbaseTransactionStatement;
+import liquibase.statement.SqlStatement;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+class CouchbaseExecutorTest {
+
+    private final CouchbaseLiquibaseDatabase database = mock(CouchbaseLiquibaseDatabase.class);
+    private final Cluster cluster = mock(Cluster.class);
+    private final CouchbaseConnection connection = mock(CouchbaseConnection.class);
+    private final TransactionExecutorService transactionExecutorService = mock(TransactionExecutorService.class);
+
+    private CouchbaseExecutor couchbaseExecutor;
+
+    @BeforeEach
+    void setUp() {
```

Comment:
Minor: Maybe this could be `@BeforeAll`?

Reply:
@BeforeAll must be static. I don't have any static logic here, so I don't think so

Reply:
Why we need initialize it every time if you mock database 1 time? What's the point of it? Isn't it simpler to init 1 time also? You can use @Mock and @InjectMocks. InjectMocks should work with constructor injection or setter injection also. So I propose to try this annotations and mock and init objects 1 time if it's possible.

But you can leave mock logic below as it is in before each.
```
        when(database.getConnection()).thenReturn(connection);
        when(connection.getTransactionExecutorService()).thenReturn(transactionExecutorService);
        when(connection.getCluster()).thenReturn(cluster);
```

Reply:
Are you guys seriously arguing what's better `@Mock `or `mock(..)` and `@Beforeach`/` @Beforeall ` in unit test?
Pick whatever you want there is no big difference 😄